CHANGELOG.md

```diff
@@ -554,8 +554,8 @@
 <a name="v0.0.1"></a>
 ## v0.0.1 - 2020-06-24
 
-[Unreleased]: https://github.com/grafana/synthetic-monitoring-agent/compare/v0.20.2...HEAD
-[v0.20.2]: https://github.com/grafana/synthetic-monitoring-agent/compare/v0.20.1...v0.20.2
+[Unreleased]: https://github.com/grafana/synthetic-monitoring-agent/compare/v0.21.0...HEAD
+[v0.21.0]: https://github.com/grafana/synthetic-monitoring-agent/compare/v0.20.1...v0.21.0
```

Comment:
Do you mean to overwrite `0.20.2` instead of adding another entry for `0.21.0`?

Reply:
I reckon the changelog might look a little bit weird, but because the previous releases were not "fully released", there is not a tag for `v0.20.2`, this new release will include previous changes. The reason to bump the minor version is due to the protobuf definitions, which even if is a small change, it matches better with a minor update than a patch.
See [`/scripts/release`](https://github.com/grafana/synthetic-monitoring-agent/blob/main/scripts/release) for more info on the release process.

Reply:
Thanks!